packages/nx/src/plugins/js/package-json/create-package-json.ts

```diff
@@ -181,6 +181,7 @@ export function findProjectsNpmDependencies(
   target: string,
   options: {
     helperDependencies?: string[];
+    ignoredDependencies?: string[];
```

Comment:
I think it's better to delete this after the fact...

I also do not like the fact that `helperDependencies` is here.

```ts
const ignoredDependencies = ['react'];
const packageJson = createPackageJson(...);
for (const ignoredDep of ignoredDependencies) {
  delete packageJson.dependencies[ignoredDep];
  delete packageJson.devDependencies[ignoredDep];
  delete packageJson.peerDependencies[ignoredDep];
}
```

Reply:
Okay, it wouldn't work for transitive dependencies. I guess this is fine.